spec/requests/policies_spec.rb

```diff
@@ -98,6 +98,36 @@ def test_policy_profile_query(object, api_object_policy_profiles_url)
     expect_result_resources_to_include_data("resources", "guid" => Array.wrap(ps2.guid))
   end
 
+  def test_policy_profile_policies_query(object, api_object_policy_profiles_url)
+    api_basic_authorize
+
+    object.add_policy(ps1)
+    object.add_policy(ps2)
+
+    get api_object_policy_profiles_url, :params => { :expand => "resources", :attributes => "miq_policies" }
+
+    expect_query_result(:policy_profiles, 2)
+    expect(response.parsed_body).to include("name" => "policy_profiles")
+    # Alberto, I wasn't able to get this working
+    # expected = a_hash_including(
+    #   "name" => "policy_profiles",
+    #   "resources" => a_collection_including(
+    #     a_hash_including(
+    #       "guid" => ps2.guid,
+    #       "miq_policies" => a_collection_including(
+    #         a_hash_including(
+    #           "guid" => Array.wrap([p1.guid, p2.guid])
+    #         )
+    #       )
+    #     )
+    #   )
+    # )
+    # expect(response.parsed_body).to include(expected)
+    # So I instead went with this -
```

Comment:
The following seems to work, give it a whirl ....

```
    expected = {
      "name"  => "policy_profiles",
      "count" => 2,
      "resources" => a_collection_containing_exactly(
        a_hash_including(
          "guid" => ps1.guid,
          "miq_policies" => a_collection_containing_exactly(
            a_hash_including("guid" => p1.guid),
            a_hash_including("guid" => p2.guid)
          )
        ),
        a_hash_including(
          "guid" => ps2.guid,
          "miq_policies" => a_collection_containing_exactly(
            a_hash_including("guid" => p3.guid)
          )
        )
      )
    }
    expect(response.parsed_body).to include(expected)
```

Reply:
Thanks @abellotti that works!

Reply:
No problem.  Maybe just update the alignment there for Mr. Rubocop 🤖  Thanks.